scripts/fix_lotus_encumbrances/fix_lotus_encumbrances.py

```diff
@@ -120,13 +121,33 @@ def get_fiscal_year_ids_by_query(query):
     return ids
 
 
-def get_order_ids_by_query(query):
-    params = {'query': query, 'offset': '0', 'limit': ITEM_MAX}
-    try:
-        r = requests.get(okapi_url + 'orders-storage/purchase-orders', headers=headers, params=params)
+def get_by_chunks(url, query, key):
```

Comment:
Can we refactor this method, first request woud be to retieve totalCount, we can use for offset limit values 0 or 1 (need to check what validation allows), after this we need to build requests based on this totalCound and send them simultaniously to speed-up performance of this operation. 
We ca use 5 as concurrent requests count based on this constant: https://github.com/folio-org/mod-orders/blob/master/src/main/java/org/folio/rest/RestConstants.java#L26

Reply:
@SerhiiNosko We could optimize this for speed as you suggest, but I don't think it's worth the trouble. Getting 10000 orders takes 0.5s, compared to 182s for the rest of the code. This is probably why this part was not optimized earlier when the code was made async with parallel requests.

Reply:
okay, I am okay with this, thanks